src/js/common/schemaform/fields/ArrayField.jsx

```diff
@@ -136,8 +136,9 @@ export default class ArrayField extends React.Component {
           ? false
           : val;
       });
+      const editingState = this.props.uiSchema['ui:options'].reviewMode;
       const newState = _.assign(this.state, {
-        editing: newEditing.concat(false)
+        editing: newEditing.concat(editingState)
```

Comment:
What problem does this solve?

Reply:
Editing state is now configurable, as this view needs to display the items as not editing, vs editing, by default.